apps/admin-gui/src/app/vos/components/application-detail/application-detail.component.html

```diff
@@ -142,15 +142,17 @@ <h2 class="page-subtitle">
       <ng-container matColumnDef="value">
         <th *matHeaderCellDef mat-header-cell></th>
         <td *matCellDef="let userData" class="word-break-all" mat-cell>
-          {{userData.value}}
-          <button
-            *ngIf="(application.state === 'NEW' || application.state === 'VERIFIED') &&
+          <div class="d-flex align-items-center">
```

Comment:
I haven't seen it before but could you please remove also `mb-2` (bottom margin) class for the `Re-send notification` button at this page? (line 42)
![image](https://user-images.githubusercontent.com/70371638/220954829-2219a57e-34a9-43e8-a854-c9ff3885b625.png)